schema.graphql

```diff
@@ -552,12 +552,43 @@ type Transfer @entity(immutable: true) {
   from: Bytes!
 }
 
+type Dependency @entity {
+  id: ID!
+  preferredCDN: String!
+  additionalCDNs: [DependencyAdditionalCDN!]! @derivedFrom(field: "dependency")
+  additionalCDNCount: BigInt!
+  preferredRepository: String!
+  additionalRepositoryCount: BigInt!
+  additionalRepositories: [DependencyAdditionalRepository!]! @derivedFrom(field: "dependency")
+  scripts: [DependencyScript!]! @derivedFrom(field: "dependency")
```

Comment:
do we also want a `script` field here (analogous to `Project.script`)

Reply:
Resolved in https://github.com/ArtBlocks/artblocks-subgraph/pull/159